validation/helpers.py

```diff
@@ -179,20 +179,29 @@ def get_translations(results):
     matches = []
 
     for i in results["results"]:
-        translations, analysis = extract_translations(i["lemma_wordform"])
-        if {"translations": translations, "analysis": analysis} not in matches:
-            matches.append({"translations": translations, "analysis": analysis})
+        translations, analysis, sources = extract_translations(i["lemma_wordform"])
+        for (j, translation) in enumerate(translations):
+            source = sources[j]
```

Comment:
Are `translations` and `sources` parallel arrays? In that case, `zip()` _might_ be cleaner:

```suggestion
        for translation, source in zip(translations, sources):
```

Reply:
Nice, thanks!